common/tools/cointool.py

```diff
@@ -857,9 +910,9 @@ def render(paths, outfile, verbose, bitcoin_only):
     for key, value in support_info.items():
         support_info[key] = Munch(value)
 
-    def do_render(src, dst):
+    def do_render(src: str, dst: TextIO) -> None:
         if verbose:
-            click.echo(f"Rendering {src} => {dst}")
+            click.echo(f"Rendering {src} => {dst.name}")
```

Comment:
does this typecheck? `name` doesn't _sound_ like something on `TextIO`, but maybe it's there?